lib/cisco_node_utils/feature.rb

```diff
@@ -19,24 +19,69 @@
 module Cisco
   # Feature - node util class for managing common features
   class Feature < NodeUtil
-    def self.nv_overlay_enabled?
-      config_get('feature', 'nv_overlay')
+    # Note that in most cases the enable methods should only enable;
+    # however, for test purposes it is sometimes convenient to support
+    # feature disablement for cleanup purposes.
+    # ---------------------------
+    def self.bgp_enable(state='')
+      config_set('feature', 'bgp', state: state)
+    end
+
+    def self.bgp_enabled?
+      config_get('feature', 'bgp')
     rescue Cisco::CliError => e
-      # cmd will syntax when feature is not enabled.
+      # cmd will syntax reject when feature is not enabled
```

Comment:
It's a cut&paste from the old code which was doing show run bgp. I'll remove it.